guide/16-geoenrichment-guides/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<p>Mohit, a question here: Shouldn't the line connecting from "get_countries()" and "Country" be dashed line? instead of the solid line??</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>this is fixed.</p>